### PR TITLE
4423menu

### PR DESCRIPTION
**Pull request by felixgondwe**
protein and regulation tab menus

### CODE REVIEW

File: src/sgd/frontend/yeastgenome/views/locus_views.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 TEMPLATE_ROOT = 'src:sgd/frontend/yeastgenome/static/templates/'
 
 def get_locus_obj(identifier):
-    backend_locus_url = config.backend_url + '/locus/' + identifier
+    #temp addition: overview
+    backend_locus_url = config.backend_url + '/locus/' + identifier + '/overview'
```
Comment (ragingsquirrel3) [inline src/sgd/frontend/yeastgenome/views/locus_views.py]: remove the '/overview'

Comment (felixgondwe) [inline src/sgd/frontend/yeastgenome/views/locus_views.py]: done

File: client/jsx/views/regulation_view.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const React = require("react");
+const ReactDOM = require("react-dom");
+const _ = require("underscore");
+const NavBar = require("../components/widgets/navbar.jsx");
+const TabsModel = require("../models/tabs_model.jsx");
+
+var regulationView = {};
+regulationView.render = function(){
+    debugger
```
Comment (ragingsquirrel3) [inline client/jsx/views/regulation_view.jsx]: please remove debugger

File: client/jsx/views/protein_view.jsx [hunk 3]
```diff
@@ -23,8 +39,10 @@ proteinView.render = function (rawDomainData, locusLength, colorScale) {
     return d;
   });
 
-  ReactDOM.render(<ProteinViewer data={domainData} locusData={locusData} colorScale={colorScale}/>
-  , document.getElementById("domain_chart"));
+
+ /* ReactDOM.render(<ProteinViewer data={domainData} locusData={locusData} colorScale={colorScale}/>
```
Comment (ragingsquirrel3) [inline client/jsx/views/protein_view.jsx]: uncomment this

File: src/sgd/frontend/yeastgenome/static/templates/regulation_details.jinja2 [hunk 4]
```diff
@@ -21,26 +21,10 @@
     <!-- Navbar
     ================================================== -->
     <!-- Includes navbar.html into template -->
-    {% include "navbar.jinja2" %}
-
+    {#{% include "navbar.jinja2" %}#}
```
Comment (ragingsquirrel3) [inline src/sgd/frontend/yeastgenome/static/templates/regulation_details.jinja2]: please remove comment

**Pull request merged**